build/NuSpecs/MUXControls.nuspec

```diff
@@ -28,6 +28,8 @@
     <file target="lib\uap10.0\Microsoft.UI.Xaml\Themes" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.UI.Xaml\Generic.xaml"/>
     <file target="lib\uap10.0\Design" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.UI.Xaml.Design\Microsoft.UI.Xaml.design.dll"/>
 
+    <file target="lib\net8.0-windows10.0.22621.0\Microsoft.UI.Xaml.Projection.dll" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.UI.Xaml.Projection\Microsoft.UI.Xaml.Projection.dll" />
```

Comment:
We need to pack a few more things here. This is what the package looks like now:

![image](https://github.com/user-attachments/assets/df335281-a349-4556-b064-87ef72618078)

We also need:
- The Generic.xaml file, in the same subfolder but under the new TFM
- The generated .xml file for the projections .dll

Not entirely sure what that Design.dll file is? Is it for the XAML Designer? I assume we need that one too?